src/app/mandate/mandate.component.html

```diff
@@ -0,0 +1,146 @@
+<main class="b-container">
+  @if (complete) {
+    <div id="banner-thank-you"></div>
+  }
+
+<!--  @if (!complete && !noAccess && !timedOut) {-->
+<!--    <mat-spinner class="complete-spinner" color="primary"-->
+<!--                 diameter="30" aria-label="Loading donation details"></mat-spinner>-->
+<!--  }-->
+
+<!--  @if (isDataLoaded) {-->
+    <div>
+      @if (false) {
+        <p class="error">
+          Sorry, you don't have access to this page. To keep your information secure we only allow viewing this
+          page temporarily after making a donation. To see donation details please check your email for a donation
+          receipt.</p>
+        <p>If you have set a donor account password you may also
+          view your <a href="/my-account/donations">donation history</a>.
+        </p>
+      }
+      @if (false) {
+        <p class="error" aria-live="assertive">
+          Your donation was not processed immediately &ndash; it may be that there is a short delay because we're very
+          busy. Please refresh this page to see the latest status. We'll also email your receipt within a few minutes when
+          your donation is processed.
+        </p>
+      }
+      <div aria-live="polite">
+        @if (complete) {
+          <div>
+            <div class="screenreader-only">
+              Thank you! Your generous regular donation has been set up.
+            </div>
+          </div>
+        }
+      </div>
+      @if (complete) {
+        <div>
+          <div class="thank-you">
+            <h2>Thank you!</h2>
+            <p class="thank-you-text">Your generous regular donation has been set up.</p>
+          </div>
+          <hr aria-hidden="true">
+          <div class="donation-summary">
+<!--            <p><strong>Hi {{ donation.firstName }},</strong></p>-->
+            <p><strong>Your donation summary</strong></p>
+            <p>Your donation of
+              <strong class="figure">{{ mandate.amount.amountInPence + ' ' + mandate.amount.currency }}</strong>
+              is worth
+              <strong class="figure">{{ totalValue + ' ' + mandate.amount.currency }}
```

Comment:
I don't think we can say it's worth double each month - its only going to be doubled for the first three months. Not sure what the best way to express that is.

And if possible IMHO we should avoid hard-coding the fact that it will be doubled for the first three months in the template and make that info somehow part of the mandate object that we will load from matchbot. Not sure exactly how we'd write it or how we'd put it in the object.

If we can work out a wording maybe hard code it in the template but with a very obvious todo comment that's visible in the browser so we won't forget to replace with actual data.